index.html

```diff
@@ -26,37 +26,321 @@ <h2 class="bottom_color">I'm a software developer </h2>
+                        <a href="#" class="linkedin">
                             <img src="images/Linkedin icon.png" alt="Linkedin icon">
                         </a>
                     </li>
                     <li>
-                        <a href="#" class="social-icon">
+                        <a href="#" class="medium">
                             <img src="images/medium.png" alt="medium icon">
                         </a>
                     </li>
                     <li>
-                        <a href="#" class="social-icon">
+                        <a href="#" class="github">
                             <img src="images/github.png" alt="github icon">
                         </a>
                     </li>
                     <li>
-                        <a href="#" class="social-icon">
+                        <a href="#" class="angellist">
                             <img src="images/angellist.png" alt="angellist icon">
                         </a>
                     </li> 
                 </ul>
                 
             </div>
+            <div class="action">
+
+                <img class="normal_button" src="images/Disabled.png" alt="Scroll Bar">
+    
+            </div>
             
         </section>
         
+        <section class="recent_works">
+            <div class="primary_txt">
+                <h2 >My Recent Works</h2>
+                <img class="indicator" src="images/Indicator.png" alt="Indicator">
+
+            </div>
+            <div class="card_Work_Row_1">
+                <div class="card">
+                    <div class="image_placeholder_card">
+                       
+                    <div class="text">
+                       
+                        <div>
+                        <p class="project_title">Multi-Post Stories Gain+Glory</p>
+                    </div>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li class="tag1">Ruby on rails</li>
+                            <li class="tag2">css</li>
+                            <li class="tag3">JavScript </li>
+                            <li class="tag4">html</li>
+                        </ul>                                                  
+                    </div>
+                        <div class="action1">
+                            <button type="button" class="button"><p class="btx1">See Project</p></button>
+                        </div>
+                        
+                         </div>
+                         
+                           
+
+                        
+                    </div>
+
+                </div>
+                <div class="card">
+                    <div class="image_placeholder_card">
+                       
+                    <div class="text">
+                       
+                        <div>
+                        <p class="project_title">Multi-Post Stories Gain+Glory</p>
```

Comment:
- [ ] Kindly I recommend to you use  a header tag `hx` the project title instead of `p` tag ,This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.